app/models/notify_user/apns.rb

```diff
@@ -1,37 +1,28 @@
 module NotifyUser
   class Apns
-    SYMBOL_NAMES_SIZE = 10 
+    SYMBOL_NAMES_SIZE = 10
     PAYLOAD_LIMIT = 255
```

Comment:
Apple have upgrade their payload size to 2 kilobytes. I'm not quite sure what that translates to in character limit.